itwm_example/mco/optimizers/optimizers.py

```diff
@@ -285,19 +367,26 @@ def optimize(self):
         f = MultiobjectiveFunction(
             multiobjective_function=self._score, upper_bounds=upper_bounds
         )
-        instrumentation = self._create_instrumentation()
+        instrumentation = self._assemble_instrumentation()
         instrumentation.random_state.seed(12)
         ng_optimizer = ng.optimizers.registry[self.algorithms](
             instrumentation=instrumentation, budget=self.budget
         )
         for _ in range(ng_optimizer.budget):
             x = ng_optimizer.ask()
             value = f.multiobjective_function(x.args)
-            volume = f.compute_aggregate_loss(value, *x.args, **x.kwargs)
+            volume = f.compute_aggregate_loss(
+                self._swap_minmax_kpivalues(value), *x.args, **x.kwargs
+            )
             ng_optimizer.tell(x, volume)
 
-        for point, value in f._points:
-            yield point[0], value, [1] * len(self.kpis)
+            if self.verbose_run:
+                yield x.args, value, [1] * len(self.kpis)
+
+        if not self.verbose_run:
+            for point, value in f._points:
+                value = self._swap_minmax_kpivalues(value)
+                yield point[0], value, [1] * len(self.kpis)
```

Comment:
I think the only code coverage gap is testing `verbose_run` - if we can easily put a unit test for these two outcomes it would be beneficial to do so here